src/diffusers/models/unets/unet_2d.py

```diff
@@ -17,10 +17,10 @@
 import torch
```

Comment:
Let's make sure we also keep a `src/diffusers/models/unet_2d.py` file for backward compatibility

Reply:
Done in a7adbeaa